### ISSUE TITLE
data.aws_arn: Fails to validate managed IAM policy ARN

### ISSUE THREAD

**Issue by phyber** [terraform-providers/terraform-provider-aws#11449]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.12.18
+ provider.aws v2.43.0
```

### Affected Resource(s)

* `data.aws_arn`

### Terraform Configuration Files

```hcl
data aws_arn cloudwatch_readonly_access {
  arn = "arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess"
}
```

### Debug Output

N/A

### Panic Output

N/A

### Expected Behavior

I should have been able to use a managed IAM policy ARN in the `aws_arn` data type.

### Actual Behavior

The following errors occurs

```
Error: "arn" doesn't look like a valid ARN ("^arn:[\\w-]+:([a-zA-Z0-9\\-])+:([a-z]{2}-(gov-)?[a-z]+-\\d{1})?:(\\d{12})?:(.*)$"): "arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess"
```

### Steps to Reproduce

1. Create some Terraform including the above data source
2. `terraform plan`

### Important Factoids

N/A

### References

N/A

### Other

I'll have a PR shortly to fix this issue.

**Comment by phyber** [comment]
I have submitted a pull request to solve this issue. (Just leaving this comment since the community note says to do so, PR already linked against issue).

**Comment by bflad** [comment]
The fix for this (including the `aws_arn` data source and other places where use `validateArn` for attribute validation) has been merged and will release with version 2.44.0 of the Terraform AWS Provider, next Thursday. Thanks @phyber for the implementation. 👍
